5m5v-bot.js

```diff
@@ -23,25 +23,40 @@ if (!config.users.every(user => ['language', 'email', 'username', 'password'].ev
 
 const languages = config.users.map(user => user.language);
 const pollingIntervalMs = 40 * 1000;
+const loginDelayMs = 5 * 60 * 1000;
 const retweetDelayMs = config.delaytime || 2 * 60 * 1000;
 const isDryRun = process.argv[2] === '--dry-run';
 const tweetFilter = new TweetFilter(config.exclude, languages);
 
-async function getApiKey(user) {
-  if (!user.apiKey) {
-    console.log(`Logging in as ${user.username}...`);
+async function getApiKey(user, { retry = true } = {}) {
+  if (user.apiKey) {
+    return user.apiKey;
+  }
 
-    user.apiKey = await new Rettiwt().auth.login(user.email, user.username, user.password);
+  while (true) {
+    try {
+      console.log(`Logging in as ${user.username}...`);
 
-    console.log('Logged in!');
-  }
+      await new Promise(resolve => setTimeout(resolve, loginDelayMs));
+
```

Comment:
I'm not sure if I remember the issue you had with the docker container crashing right ,but if it's just the script throwing wouldn't it be enough to wait before the throw. Otherwise the runtime of one circle is gonna increase a lot here

Reply:
That was mostly out of paranoia after getting the account locked. I'm thinking if the container still somehow stops or the app crashes for any other reasons, it would still potentially spam the login endpoint. I could remove the `restart: always` parameter from the docker command though.

However, this delay would only occur the first time the bot is launched or if API key becomes invalid, so I think it's fine.